### DISCUSSION TITLE
[Feature] Option to keep screen awake while uploading on iOS

### DISCUSSION THREAD

**Discussion by izzoa** [immich-app/immich#9965] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Mobile uploads will stop when the screen turns off on iOS (OS restriction). Requesting an option to forcibly keep the screen on while uploading, similar to other apps within similar functionality, so that immich can continue to upload long-running batches of photos. 

Thanks and keep up the great work!! :) 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8217
